CHANGELOG.rst

```diff
@@ -10,6 +10,11 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 `Unreleased`_
 -------------
 
+Added
+~~~~~
+
+* Added ``AdvertisementServiceData`` in BLEDevice.
```

Comment:
Please add something to say that this is just for macOS here.

Reply:
done